### PR TITLE
chore: redesign Playback feature

### LINKED ISSUE #84: Need to re-design Playback feature
1. Buttons next and back should be inside message box. At the beginning of the chat back button should be disabled; at the end - next button should be disabled
2. The message box in Playback should be the same size as in new chat.
3. Stop button should have another icon and should be moved to chat header

**Expected design**: 
image

Please see Figma here (available for DIAL dev team only): https://www.figma.com/file/lwL5jneveZczAj5RLcLFH2/DIAL-Chat?type=design&node-id=2889-8559&mode=design&t=i4XpbdbFspb5xHrH-4

### PR DESCRIPTION
**Pull request by IlyaBondar**
### Description

Issue #84: Need to re-design Playback feature

### CODE REVIEW

**Comment by IlyaBondar** [comment]
/deploy-review
>Deployment status: success

File: src/components/Chat/ChatHeader.tsx [hunk 1]
```diff
@@ -42,6 +49,8 @@ export const ChatHeader = ({
   onClearConversation,
   onUnselectConversation,
   setShowSettings,
+  isPlayback,
+  onCancelPlaybackMode,
```
Comment (mikitabut) [inline src/components/Chat/ChatHeader.tsx]: Actually you can not expose dispatch to top, you can do this right here - in ChatHeader

File: src/components/Chat/ChatHeader.tsx [hunk 2]
```diff
@@ -42,6 +49,8 @@ export const ChatHeader = ({
   onClearConversation,
   onUnselectConversation,
   setShowSettings,
+  isPlayback,
```
Comment (mikitabut) [inline src/components/Chat/ChatHeader.tsx]: You can get this from store right inside this component

File: src/components/Chat/ChatHeader.tsx (hunk 1 above)
Comment (IlyaBondar) [reply src/components/Chat/ChatHeader.tsx]: Done

File: src/components/Chat/ChatHeader.tsx (hunk 2 above)
Comment (IlyaBondar) [reply src/components/Chat/ChatHeader.tsx]: Done

**Comment by IlyaBondar** [comment]
/deploy-review
>Deployment status: success

**Comment by IlyaBondar** [comment]
/deploy-review
>Deployment status: success

File: package.json [hunk 3]
```diff
@@ -13,7 +13,8 @@
     "lint:fix": "eslint src --fix && prettier --write src",
     "format": "prettier --write src",
     "test": "vitest",
-    "coverage": "vitest run --coverage",
+    "coverage": "vitest run --coverage.enabled --coverage.all",
+    "coverage:covered": "vitest run --coverage",
```
Comment (mikitabut) [inline package.json:17]: what this command mean?

File: src/components/Chat/ChatMessage.tsx [hunk 4]
```diff
@@ -179,7 +179,7 @@ export const ChatMessage: FC<Props> = memo(
 
     return (
       <div
-        className={`group h-full md:px-4 ${
+        className={`group h-full min-h-[90px] md:px-4 ${
```
Comment (mikitabut) [inline src/components/Chat/ChatMessage.tsx:182]: Why do you need this?

**Review by mikitabut — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (IlyaBondar) [reply package.json:17]: "coverage" - show coverage for all typescript files ("all" option)
"coverage:covered" - only show coverage for files with tests (useful for development)

File: src/components/Chat/ChatMessage.tsx (hunk 4 above)
Comment (IlyaBondar) [reply src/components/Chat/ChatMessage.tsx:182]: minimal height equals message with one line

**Pull request merged**